src/neo-vm/Types/InteropInterface.cs

```diff
@@ -34,5 +34,10 @@ public override byte[] GetByteArray()
         {
             return _object as T;
         }
+        
+        public object GetInterface()
+        {
+            return _object;
+        }
```

Comment:
The little I know of C# makes me wonder if it will complain on returning a private member in a public function... isn't this an issue? For C++ that's fine :)

Reply:
Another thing, pay attention to not deserialize this object to byte[]... some time ago I noticed that it would be catastrophic to be able to serialize/deserialize such interfaces.